modules/ROOT/pages/mule-app-dev.adoc

```diff
@@ -0,0 +1,71 @@
+= Mule Application Development
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Mule applications enable you to perform data integrations for your use cases.
```

Comment:
I would just say "integrations" since there are other use cases that you can implement that are not data integration.